.github/workflows/check.yml

```
@@ -22,6 +22,14 @@ jobs:
         uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8
       - name: Build
         run: make build
+  licenses:
+    name: Licenses
+    runs-on: ubuntu-latest
+    steps:
```

As this requires building the Docker image, this job should depend on the `build` job:

```suggestion
  licenses:
    name: Licenses
    runs-on: ubuntu-latest
    needs:
      - build
    steps:
```